src/mixins/test/wordWrap.test.js

```diff
@@ -0,0 +1,14 @@
+import wordWrap from '../wordWrap'
+
+describe('wordWrap', function() {
+
+  it('should accept other values', function() {
+    expect(wordWrap('break-all')).toMatchSnapshot()
+  })
+
+  it('should default wrap to break-word', function() {
+    expect(wordWrap()).toMatchSnapshot()
+  })
```

Comment:
We should be able to do that, we have to do some customization to the docs template anyway before launch. 

Using spreads also solves my above comment about how to handle adding the rules to existing style blocks. I think it is more than safe to assume folks will be using babel with this anyway.